homeworks/szabolcsgesztesi/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,45 @@
+# Választási lehetőségek és a kezdő pontok
+options = ["rock", "paper", "scissors"]
+player1_score = 0
+player2_score = 0
+
+# Körök számának bekérése
+while True:
+    chosen_rounds = int(input("Hány kört szeretnél játszani? "))
+    if chosen_rounds % 2 != 0:
+        print("---- Kezdődhet a játék. -----")
+        break
+    else:
+        print("Páratlan körrel indul csak a játék.")
+
+# A kiválasztott számnak megfelelő körök lejátszása:
+for rounds in range(chosen_rounds):
+
+    # Mindkét játékos választ
+    while True:
+      player_1_turn = input("Első játékos választ: ")
+      if player_1_turn in options:
+           break
+
+    while True:
+      player_2_turn = input("Második játékos választ: ")
+      if player_2_turn in options:
+           break
+    
+    # Játékszabály érvényesítése
+    if player_1_turn == player_2_turn:
```

Comment:
Szuper hogy lekezeled azt, hogy ha ugyanazt a jelet választják, de így ez a kör teljesen kimarad, és senki nem kap pontot, ezáltal lehetséges lesz a döntetlen. Ezt a döntetlent viszont nem kezeled a 41. soron kezdődő if-else-ben. Így pl előfordulhat ilyen végkimenetel:

```
Végeredmény: Első játékos 0 pont, Második játékos 0 pont
Második játékos nyert!

```
Mivel csak azt nézed meg hogy a player1_score nagyobb e mint a player2_score, ha ez nem igaz, döntetlen esetén is a második játékos lesz kihozva győztesnek.

A legjobb megoldás az lenne, ha lenne egy while loop-od minden egyes körre, és addig nem engedne át a következő körre, amíg a két játékos eltérő választ nem ad. Így kizárhatod a döntetlent.